apps/mobile/src/screens/CreationOptions/CreationOptions.js

```diff
@@ -6,52 +6,48 @@ import { openURL } from 'hooks/useOpenURL'
 
 export default function CreationOptions () {
   const { t } = useTranslation()
-  console.log('DOES THIS EVERM HAAHHAPEN?')
   const handleCreatePost = () => {
     openURL('/create/post')
   }
-
   const handleCreateGroup = () => {
     openURL('/create/group')
   }
 
   return (
-    <View className='flex-1 bg-background p-4'>
```

Comment:
It is easy in React Native, not unlike web, to end-up with an extra View wrapper where it is unnecessary. It has little impact, but in my experience it can tend to become a source of unnecessary complexity over time with styling and such. So always confirm you need the extra container, most often "container" containers are unnecessary.